Index: ../ClassList/app/src/main/java/com/example/classlist/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.classlist;\r\n\r\nimport androidx.appcompat.app.AppCompatActivity;\r\n\r\nimport android.os.Bundle;\r\n\r\npublic class MainActivity extends AppCompatActivity {\r\n\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        setContentView(R.layout.activity_main);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../ClassList/app/src/main/java/com/example/classlist/MainActivity.java b/../ClassList/app/src/main/java/com/example/classlist/MainActivity.java
--- a/../ClassList/app/src/main/java/com/example/classlist/MainActivity.java	(revision 3f7b12a113723f34ec2812600cd6b98180479c5d)
+++ b/../ClassList/app/src/main/java/com/example/classlist/MainActivity.java	(date 1648519462593)
@@ -11,4 +11,14 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+
+    /*
+
+import static userController.getUserByName;
+User user = getUserByName("testUser");
+user.getAuthenticationData();
+
+
+
+     */
 }
\ No newline at end of file
